### PR TITLE
Add overlay for HUAWEI-PIC-LX9

### PR DESCRIPTION
**Pull request by exzemat**
Hello,
just added power-profil from extracted /product/etc/power_profile.xml.
I use config.xml from generic huawei : work ok

### CODE REVIEW

**Comment by phhusson** [comment]
You need to add it to overlay.mk as well

**Comment by exzemat** [comment]
Sorry (gitub noobs),
I don't understand.
I need to pull request in other branch or do other modification ?

Apparemment nous sommes tous les deux français, donc je me permets de doubler mon message pour être sur de comprendre/être compris...
Bonjour (gitub noobs),
Je n'ai pas bien compris, je dois modifier autre chose dans le code ou c'est juste que je dois modifier ma requête git en poussant sur overlay.mk aussi (ou à la place de vendor hardware overlay) ?
Merci,

**Comment by exzemat** [comment]
overlay.mk added now...

File: Huawei/PIC-LX9/Android.mk [hunk 1]
```diff
@@ -0,0 +1,8 @@
+LOCAL_PATH := $(call my-dir)
+include $(CLEAR_VARS)
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := treble-overlay-huawei-PIC
```
Comment (phhusson) [inline Huawei/PIC-LX9/Android.mk]: Please give it (and folder) the commercial name.
I guess it's Huawei Nova 2 according to google?

File: Huawei/PIC-LX9/res/values/config.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <integer-array name="config_autoBrightnessLevels">
+        <item>8</item>
+        <item>55</item>
+        <item>350</item>
+        <item>1600</item>
+        <item>2550</item>
+    </integer-array>
+    
+    <bool name="config_suspendWhenScreenOffDueToProximity">false</bool>
+    <bool name="config_supportSystemNavigationKeys">false</bool>
```
Comment (phhusson) [inline Huawei/PIC-LX9/res/values/config.xml]: Uh that's weird, all Huawei devices i know have it. You're sure of this?

This option is about opening notifications using fingerprint gestures.

File: Huawei/PIC-LX9/Android.mk (hunk 1 above)
Comment (exzemat) [reply Huawei/PIC-LX9/Android.mk]: yes, it's nova2

File: Huawei/PIC-LX9/res/values/config.xml (hunk 2 above)
Comment (exzemat) [reply Huawei/PIC-LX9/res/values/config.xml]: this values comes from /framework-res/res/values.
I test today with "true" : no change (opening notifications using fingerprint gestures no work)

Comment (exzemat) [reply Huawei/PIC-LX9/res/values/config.xml]: original /framework-res/res/values
[http://pix.toile-libre.org/upload/original/1577035521.png](url)

Comment (phhusson) [reply Huawei/PIC-LX9/res/values/config.xml]: You enabled fingerprint gestures in treble settings, huawei section?

Comment (exzemat) [reply Huawei/PIC-LX9/res/values/config.xml]: yes.
(my last edit) : phh Treble Settings/Huawei features/enable fingerprint gestures : active

Comment (exzemat) [reply Huawei/PIC-LX9/res/values/config.xml]: no work, so I remove this 2 options

**Pull request merged**